docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -91,19 +91,40 @@ This analysis can be categorised as relating
-  * replica exchange overlap matrix (for repex only)
-  * timeseries of replica states
-  * forward and reverse estimates of free energies
-
-* relating to structural analysis. If a protein was present, these analyses first center and align the system so that
+.. todo: issue 792, consolidate this page into its own analysis page and link both RBFE and AFE pages to it
+
+* Energetic and replica exchange analysis.  These analyses consider the swapping and energetic overlap between the
+  different simulated states to try to establish the convergence and correctness of the estimate of free energy
+  difference produced.
+
+  * MBAR overlap matrix.  This plot is used to assess if the different lambda states simulated overlapped energetically.
+    Each matrix element represents the probability of a sample from a given row state being observable in a given column
+    state.
+    This plot should show that the diagonal of the matrix has some "width" so that the two end states are connected.
+  * Replica exchange overlap matrix (for repex type simulations only).  Similar to the MBAR overlap matrix, this shows
+    the probability a given lambda state exchanged with another.  Again, the diagonal of this matrix should be at least
+    tridiagonal wide for the two end states to be connected.
+  * timeseries of replica states.  This plot shows the evolution of time of the different lambda states as they are
+    exchanged between different configurations.
+    This plot should show that the states are freely mixing and that there are no cliques forming.
+  * forward and reverse estimates of free energies.  This analysis considers the free energy estimate produced using
+    increasingly larger fractions of the total data, and is repeated both forwards and backwards in time.
+    To assess convergence the forward and reverse estimates should agree within error using only a fraction of the total
+    data[2]_.
+
+* Structural analysis. If a protein was present, these analyses first center and align the system so that
   the protein is considered the frame of reference.
 
-  * ligand RMSD  This produces a plot called ``xyz.png`` and a results entry ``xyz``
-  * protein 2D RMSD
-  * ligand COM drift
+  * ligand RMSD.  This produces a plot called ``ligand_RMSD.png`` and a results entry ``ligand_RMSD`` which gives the
+    RMSD of the ligand molecule over time relative to the first frame, for each simulated state.  In correct simulations
+    this metric should quickly converge to a stable value of less than 5 angstrom.
+  * ligand COM drift.  For simulations with a protein present, this metric gives the total distance of the ligand COM
+    from its initial starting (docked) position.  If this metric increases over the course of the simulation it
+    indicates that the ligand drifted from the binding pocket, and the simulation is unreliable.
+    This produces a plot called ``ligand_COM_drift.png`` and a results entry ``ligand_COM_drift``.
```

Comment:
Is there a rough estimate (in terms of Angstrom) that should raise concerns?

Reply:
I've put 5A?

Reply:
I haven't worked much with COM analysis so I'm not sure what a reasonable, but i agree it should definitely be below 5A.

Reply:
This is very much an unknown - i.e. it's very binding site dependent and there are lots of different folks in literature claiming lots of different values. Same thing for the RMSD metric (indeed because we are using non-symmetry correct RMSD, a 5A RMSD could easily be much smaller).

I would make it clear in the text that these are example values and that your mileage will vary depending on the system you are studying.